macos.md

```diff
@@ -289,7 +289,7 @@ Make sure that you're running the 2nd-newest macOS version or the newest version
     ```
 
     It should print out a welcome message like this:<br><br>
-    <img src="macos-6-docker-hello-world.png"><br><br>
+    <img src="macos-7-docker-hello-world.png"><br><br>
```

Comment:
this change can also be pulled out into a separate PR

Reply:
https://github.com/upleveled/system-setup/pull/66